src/sider/senoppfolging/VeiledningRutine.tsx

```diff
@@ -12,8 +13,12 @@ const texts = {
     "Kontakt behandler: Ta kontakt med behandler for å innhente medisinske opplysninger",
     "AAP: Vurder om den sykmeldte bør søke om AAP",
   ],
+  link: "Les mer om behovsvurdering og § 14 a for sykmeldte her",
```

Comment:
Kanskje vi kunne korte ned enda mer og droppet `les mer`. Ser at [aksel](https://aksel.nav.no/god-praksis/artikler/lenker) skriver at man ikke trenger det. For eksempel kunne vi skrevet:
```suggestion
  link: "Behovsvurdering og § 14 a for sykmeldte",
```

Reply:
Bra artikkel! Da forkorter jeg den enda mer 👍🏼